### PR TITLE
Add date to 2.0 release notes

### PR DESCRIPTION
**Pull request by chr11115**

### CODE REVIEW

**Review by mistafunk — CHANGES_REQUESTED** [review]
* Add new section "Software Requirements" below "Installation"
  * Add "Windows 8.1, 10 and 11 (64bit)"
  * Add CityEngine 2021.1 or older
  * Add "Autodesk Maya 2019, 2020 or 2022"
* Update System Requirements in Development section
  * Limit to Maya 2019-2022
* Update "examples" link in ToC to directly go to https://esri.github.io/cityengine/serlio#examples

File: README.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ External documentation:
 Please refer to the [release notes](#release-notes) for the supported CityEngine version.
 ### Installation
 
+#### Software Requirements
+- Windows 8.1, 10 and 11 (64bit)
```
Comment (mistafunk) [inline README.md:38]: My bad, Windows 8.1 is not supported by Maya 2022 - let's remove it altogether.

**Review by mistafunk — CHANGES_REQUESTED** [review]
Just one tiny change left, afterwards feel free to merge.

File: README.md (hunk 1 above)
Comment (chr11115) [reply README.md]: removed.

**Review by mistafunk — APPROVED** [review]
Let's go.

**Pull request merged**